packages/tanstack-react-query/test/__helpers.tsx

```diff
@@ -0,0 +1,115 @@
+import '@testing-library/jest-dom/vitest';
+import { routerToServerAndClientNew } from '../../tests/server/___testHelpers';
```

Comment:
I am not sure we should be doing this - maybe we need a `@trpc/test`-package

I noticed that vitest's watcher behaved a bit funny when I was debugging subscription issue yesterday, unsure if related but thinking it could be